### DISCUSSION TITLE
⭐Starstruck Exchange ⭐

### DISCUSSION THREAD

**Discussion by itspasindu** [Schweinepriester/github-profile-achievements#693] [General]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Comment by alireid** [comment]
Hi!

**Reply by itspasindu** [reply]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Comment by treeplate** [comment]
starred, how about treeplate/syllad

**Reply by PARKER-X** [reply]
> treeplate/syllad

done bro
star my repo
https://github.com/PARKER-X/BarBar
https://github.com/PARKER-X/carpido/

**Reply by treeplate** [reply]
done

**Reply by nguyen-tuan-hiep** [reply]
done, star my repo: https://github.com/nguyen-tuan-hiep/Introduction-to-Communication-Engineering

**Reply by ghost** [reply]
@treeplate @PARKER-X 
done all pls star back , https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Reply by nguyen-tuan-hiep** [reply]
done, star my repo: https://github.com/nguyen-tuan-hiep/Introduction-to-Communication-Engineering

**Reply by ghost** [reply]
done, star back :  https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰

**Reply by ghost** [reply]
done, star back :  https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

**Comment by orzklv** [comment]
swap pls, only 3 left ;D
https://github.com/katsuki-yuri/neko

**Reply by ghost** [reply]
done, star back :  https://github.com/darkhackkr/Draino-Metamask-ETH-Drainer

**Reply by orzklv** [reply]
done!

**Comment by YuZhangWang** [comment]
done
[YuZhangWang](https://github.com/YuZhangWang/YuZhangWang)
